source/_components/light.template.markdown

```diff
@@ -34,7 +35,12 @@ light:
           service: script.theater_lights_level
           data_template:
             brightness: "{{ brightness }}"
+        availability_template: >-
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.